tensorflow/tools/ci_build/Dockerfile.debian.jessie.cpu

```diff
@@ -5,7 +5,15 @@ MAINTAINER Jan Prach <jendap@google.com>
 # Copy and run the install scripts.
 COPY install/*.sh /install/
```

Comment:
for this and the next ones, are these changes that were made to 1.0.1 and need backporting?